src/components-examples/material/bottom-sheet/bottom-sheet-harness/bottom-sheet-harness-example.spec.ts

```diff
@@ -0,0 +1,71 @@
+import {TestBed, ComponentFixture, waitForAsync, inject} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatBottomSheetHarness} from '@angular/material/bottom-sheet/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatBottomSheetModule} from '@angular/material/bottom-sheet';
+import {BottomSheetHarnessExample} from './bottom-sheet-harness-example';
+import {OverlayContainer} from '@angular/cdk/overlay';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+
+describe('BottomSheetHarnessExample', () => {
+  let fixture: ComponentFixture<BottomSheetHarnessExample>;
+  let loader: HarnessLoader;
+  let overlayContainer: OverlayContainer;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatBottomSheetModule, NoopAnimationsModule],
+        declarations: [BottomSheetHarnessExample]
+      }).compileComponents();
+    }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BottomSheetHarnessExample);
+    fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+    inject([OverlayContainer], (oc: OverlayContainer) => {
+      overlayContainer = oc;
+    })();
+  });
+
+  afterEach(() => {
+    // Dismiss the bottom sheet once the tests are done. This is necessary because the
+    // "MatBottomSheet" service is not destroyed automatically by TestBed, and it could
+    // mean that bottom sheets are left in the DOM.
+    fixture.componentInstance.bottomSheet.dismiss();
+    fixture.detectChanges();
+    // Angular won't call this for us so we need to do it ourselves to avoid leaks.
+    overlayContainer.ngOnDestroy();
+    overlayContainer = null!;
+  });
```

Comment:
I think that we can omit this `afterEach` from the example, we mostly need it for our own tests.